docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -41,32 +41,11 @@ Refer to [Cloud provider regions](../release-notes#cloud-provider-regions) for a
 
 Yugabyte Cloud supports all the regions that have robust infrastructure and sufficient demand from customers. We are continuously improving region coverage, so if there are any regions you would like us to support, reach out to [Yugabyte Support](https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431).
 
-### What version of YugabyteDB does Yugabyte Cloud run on
-
-Yugabyte Cloud runs on the YugabyteDB 2.8 [stable release](../../releases/whats-new/stable-release/).
-
-To see the version you are running, connect in cloud shell using YSQL and run the following command:
-
-```sh
-yugabyte=> SELECT version();
-```
-
-```output
-                                                  version                                                   
-------------------------------------------------------------------------------------------------------------
- PostgreSQL 11.2-YB-2.8.0.0-b0 on x86_64-pc-linux-gnu, compiled by gcc (Homebrew gcc 5.5.0_4) 5.5.0, 64-bit
-(1 row)
-```
-
-### Can I test YugabyteDB locally?
-
-To test locally, [download](https://download.yugabyte.com) and install YugabyteDB on a local machine. Refer to [Quick Start](../../quick-start). For accurate comparison with cloud, be sure to download the version that is running on Yugabyte Cloud.
-
 ### What are the differences between free and standard clusters?
 
-Use the free cluster to get started with YugabyteDB. The free cluster is limited to a single node and 10GB of storage, and although not suitable for production workloads, the cluster includes enough resources to start exploring the core features available for developing applications with YugabyteDB. You can only have one free cluster.
+Use the free cluster to get started with YugabyteDB. The free cluster is limited to a single node and 10GB of storage, and although not suitable for production workloads, the cluster includes enough resources to start exploring the core features available for developing applications with YugabyteDB. Free clusters are provisioned with the Latest release version of YugabyteDB. You can only have one free cluster.
```

Comment:
Worth linking to the release notes for Latest in the second-last sentence?

Reply:
We're using the term "edge release" in cloud to distinguish from our normal latest/stable release series definitions for OSS/platform. Edge releases won't always correlate to https://docs.yugabyte.com/latest/releases/whats-new/latest-release/ (although sometimes/often it may)

Reply:
Would recommend wording to say "Free clusters are pinned to the edge release track for versioning" or something along those lines (with an eventual link into the cloud DB versioning docs that gives the definition of our available cloud release channels)